packages/rest/README.md

```diff
@@ -52,16 +52,44 @@ Application options.
 const app = new RestApplication({
   rest: {
     port: 3001,
+
+    // Optional configuration for how to serve OpenAPI spec
+    openApiSpec: {
```

Comment:
IMO, this configuration example has too many comments to make it practical. That's why I proposed to move this content to https://github.com/strongloop/loopback-next/blob/master/docs/site/Server.md, where you can split the code into multiple sections/paragraph and use regular text instead of code comments.

I am fine with your proposal to leave consolidation of README out of scope of this pull request.

What I would like to ask you is to move this _new README content_ to `Server.md` right now, so that we don't introduce more work for the future consolidation effort; plus reformat the documentation of config options into a form that's easier to read. It would be great to give a wider context to the readers while you are at this, e.g. why and when to change `openApiSpec` configuration, etc.

Reply:
I'll clean up the docs.